### PR TITLE
Use flag file to get signaled when backend is ready

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -62,6 +63,44 @@ export const Application = () => {
...
+        const channel = cockpit.channel({
+            payload: "fswatch1",
+            path: "/run/anaconda/",
+            superuser: "try",
+        });
+
+        channel.addEventListener("message", (ev, data) => {
+            const item = JSON.parse(data);
+
+            if (item.event === "created" && item.path === "/run/anaconda/backend_ready") {
```
Comment (jelly) [inline src/components/app.jsx:90]: if I recall correctly you can use `cockpit.file("/run/anaconda/backend_ready").watch()` directly on a non-existing file.

Comment (KKoukiou) [reply src/components/app.jsx:90]: I was wondering that's why I asked you to review :P ok I will check!

Comment (KKoukiou) [reply src/components/app.jsx]: Works perfectly thanks

**Comment by jkonecny12** [comment]
So I was closer with my solution than I tough. That is great!

Thanks for helping me. I'll test it out.

**Review by rvykydal — APPROVED** [review]

**Pull request merged**